schema.sql

```diff
@@ -8,38 +8,40 @@ CREATE SCHEMA web3storage
     github TEXT,
     public_address TEXT NOT NULL,
     used_storage bigint DEFAULT 0,
-    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL
+    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+    updated_at timestamp with time zone
   )
   CREATE TABLE auth_tokens (
     id BIGSERIAL PRIMARY KEY,
     name TEXT NOT NULL,
     secret TEXT NOT NULL,
     user_id bigint NOT NULL,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
-    deleted_at timestamp with time zone DEFAULT timezone('utc'::text, now())
+    deleted_at timestamp with time zone
   )
   CREATE TABLE uploads (
     id BIGSERIAL PRIMARY KEY,
     user_id bigint NOT NULL,
-    auth_token_id bigint NOT NULL,
+    auth_token_id bigint,
     content_id bigint NOT NULL,
     name TEXT,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
-    deleted_at timestamp with time zone DEFAULT timezone('utc'::text, now())
+    deleted_at timestamp with time zone
   )
   CREATE TABLE contents (
     id BIGSERIAL PRIMARY KEY,
     cid TEXT UNIQUE NOT NULL,
     dag_size bigint DEFAULT 0,
-    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL
+    inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
+    updated_at timestamp with time zone
   )
   CREATE TABLE pins (
     id BIGSERIAL PRIMARY KEY,
     content_id bigint NOT NULL,
     pin_location_id bigint NOT NULL,
     status TEXT NOT NULL,
     inserted_at timestamp with time zone DEFAULT timezone('utc'::text, now()) NOT NULL,
-    updated timestamp(6) without time zone NOT NULL
+    updated_at timestamp with time zone
```

Comment:
Added `_at` suffix and switched to _with_ time zone with for consistency. I don't know what `(6)` does but not present with other occurances so removed.

I'm not sure if this difference was intentional?